.readthedocs.yml

```diff
@@ -16,5 +16,3 @@ python:
     - requirements: requirements.txt
     - method: pip
       path: .
-      extra_requirements:
```

Comment:
I'm not following why this was removed?

Reply:
My read is that the `doc` '`extra_requirements` in `.readthedocs.yml` is the same `doc` `extra_requirements` in `setup.py`, which was just `POT`, and which is also removed in this PR. Is that wrong?

Reply:
Ohhh wait I think you're right! We will have to see if this build the docs correctly, but I bet you're right.

Which makes me feel Bad for not having a test branch.